static/extensions/MubiLop/penguinhook.js

```diff
@@ -0,0 +1,258 @@
+// Penguin Hook
+// By MubiLop from Ruby Team (https://rubyteam.tech/)
```

Comment:
rubyteam.tech lists this extension under the [Attribution-NonCommercial-ShareAlike 4.0 International](http://creativecommons.org/licenses/by-nc-sa/4.0/?ref=chooser-v1) license, which does not comply with MIT